DFC.ServiceTaxonomy.GraphSync/Startup.cs

```diff
@@ -65,6 +76,13 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IContentTypePartDefinitionDisplayDriver, GraphSyncPartSettingsDisplayDriver>();
             services.AddScoped<IDataMigration, Migrations>();
             services.AddScoped<IContentPartGraphSyncer, GraphSyncPartGraphSyncer>();
+
+            //Notifiers
+            services.AddScoped<INotifier, CustomNotifier>();
```

Comment:
It'd be cleaner to `Replace` the existing `INotifier` and `IContentDefinitionService`, so that the existing services are not returned if someone decides to use `GetServices` or inject `IEnumerable<>`.